build/PackageDiffIgnore.xml

```diff
@@ -2135,6 +2135,22 @@
 			<!-- END ListViewBase selection changes -->
 		</Methods>
 	</IgnoreSet>
+
+    <IgnoreSet baseVersion="6.2">
+        <Types>
+        </Types>
+        <Events>
+        </Events>
+        <Fields>
+        </Fields>
+        <Properties>
+        </Properties>
+        <Methods>
+            <Member fullName="System.Void Microsoft.UI.Xaml.Controls.ElementFactory.ClearBindings()" reason="Removed in Uno 6.2" />
+            <Member fullName="System.Void Microsoft.UI.Xaml.Controls.ElementFactory.RestoreBindings()" reason="Removed in Uno 6.2" />
```

Comment:
need a regex here, since the methods are code-generated on multiple classes
```suggestion
            <Member fullName="System\.Void .+\.(Clear|Restore)Bindings()" reason="Removed in Uno 6.2" />
```